static/js/components/FacultyCarousel.js

```diff
@@ -19,11 +19,15 @@ export default class FacultyCarousel extends React.Component {
       responsive:     [
         {
           breakpoint: 600,
-          settings:   "unslick"
+          settings:   {
+            slidesToShow: 1
+          }
```

Comment:
Can you tell me more about this change? Is this because of a change to the slick carousel API?

Reply:
The last update made to `react-slick` package in MM was made around 1.5 years ago. As per the settings it looked like the slides were hidden on purpose based on devices starting from 800px and below.

Reply:
Huh. That's a mystery. Even if we wanted to hide it, this isn't the right way to do it. 

Can you do a before/after test with Lighthouse, to see what the penalty is for adding these back in?

Reply:
I did run a LightHouse performance check for mobile on the page locally and here are the results before and after adding the instructors on mobile. It didn't make a lot of difference on the performance though.
Note: I ran the test locally on a replica program page from RC, and with 3 instructors. Below are the results:

**On Master(No instructors on mobile)**
<img width="784" alt="Mobile_performance_instructor_master" src="https://user-images.githubusercontent.com/34372316/123257038-16466200-d50b-11eb-8d50-4eed5afe225e.png">

**On this branch (Instructors on mobile)**
<img width="771" alt="Mobile_performance_with_instructors" src="https://user-images.githubusercontent.com/34372316/123257044-17778f00-d50b-11eb-8574-51e593bd8b55.png">